Scenes/Screens/GameOverScreen/GameOverScreen.gd

```diff
@@ -19,4 +19,4 @@ func update_label_and_bar():
 
 func _on_RestartTimer_timeout():
 	queue_free()
-	emit_signal("timeout", find_node("RespawnInTavern").pressed)
```

Comment:
Yes, I wanted to discuss it, but I see it is a small thing, so there is nothing to worry about